scripts/check_lint.sh

```diff
@@ -11,4 +11,4 @@ echo "Running $0"
 
 DOCKER_CMD=docker
 
-${DOCKER_CMD} run --rm -e GOPROXY=${GOPROXY} -v $(pwd):/opt/workspace -w /opt/workspace golangci/golangci-lint golangci-lint run
+GOPROXY=${GOPROXY} golangci-lint run
```

Comment:
This assumes that golangci-lint is installed on the target system.
(also assumes that the current version of golangci-lint on that system is appropriate).

I would use docker or gobin with a pinned version of golangci-lint.

Reply:
Done